bindings/java/src/org/sleuthkit/datamodel/Score.java

```diff
@@ -38,42 +39,47 @@
- * there are some good and some bad, then it should have a significance of MEDIUM (or LOW).
+ * significance will be NOTABLE because the car's existence is surely notable. But, if
+ * there are some good and some bad, then it should have a significance of LIKELY_NOTABLE (or LOW).
  * Any module that detects green convertibles should have the same significance. 
  * But, different modules may have different confidences based on their approach. 
- * If a module can accurately detect a green convertible, then it should have HIGH
+ * If a module can accurately detect a green convertible, then it should have NOTABLE
  * confidence. But, if a module can't differentiate a convertible from a truck or
- * green from yellow, then it should have MEDIUM confience (or lower). 
+ * green from yellow, then it should have LIKELY_NOTABLE confidence (or lower). 
  * 
  * For a more traditional example, if a file is found in a MD5 hashset of notable files, 
- * then it would get HIGH significance because the hashset contains only known notable
- * files. And it would get HIGH confidence because the MD5 calculation and lookup 
+ * then it would get NOTABLE significance because the hashset contains only known notable
+ * files. And it would get NOTABLE confidence because the MD5 calculation and lookup 
  * process are exact matches and there is no guessing.
  *
- * For a keyword hit, the significance could be MEDIUM if the word exists in both
- * good and bad contexts, but the confidence would be HIGH because we know the word 
+ * For a keyword hit, the significance could be LIKELY_NOTABLE if the word exists in both
+ * good and bad contexts, but the confidence would be NOTABLE because we know the word 
  * existed in a document. 
  * 
  * If a file is found to be on a good list, then it could have a significance of NONE
  * and then other modules could ignore it. 
- * The confidence could be HIGH if it was based on an exact match MD5 or MEDIUM if it 
- * was based on only file name and size and we aren't entirely sure what the content was.
+ * 
  */
 public class Score implements Comparable<Score> {
 
+	private static final ResourceBundle bundle = ResourceBundle.getBundle("org.sleuthkit.datamodel.Bundle");
 	/**
 	 * Indicates the relevance of an item based on the analysis result's conclusion.
      * Significance is tied to an analysis technique type. 
+	 * 
+	 * For comparing significance, the following ordering applies
+	 * 
+	 * Bad > Good > Likely Bad > Likely Good > Unknown
+	 * 
 	 */
 	public enum Significance {
 
-		NONE(0, "None"),		//< Item is Good and has no (bad) significance
-		UNKNOWN(10, "Unknown"), //< no analysis has been performed to ascertain significance.
-		LOW(20, "Low"),
-		MEDIUM(30, "Medium"), //< Suspicious.  Could be good or bad. 
-		HIGH(40, "High");	//< Bad & notable
-
+		UNKNOWN(0, bundle.getString("Significance.Unknown.text")),				// no analysis has been performed to ascertain significance.
```

Comment:
I don't think we need UNKNOWN anymore.

Reply:
We define a SCORE_UNKNOWN as new Score(Significance.UNKNOWN, Confidence.NONE)

This is returned  by getAggregateScore() for anything that does *not* have any  analysis  result or real score.